client/components/jetpack/profile-dropdown/index.tsx

```diff
@@ -70,6 +71,7 @@ const ProfileDropdown: React.FC< Props > = ( { isOpen, close } ) => {
 	const logOut = ( e: MouseEvent ) => {
 		e.preventDefault();
 		e.stopPropagation();
+		clearToken();
```

Comment:
We need to clean the OAuth token from cookies as well since the login form also stores it as in the browser cookies.